src/Logging.AzureAppServices/test/LoggerBuilderExtensionsTests.cs

```diff
@@ -24,53 +24,106 @@ public LoggerBuilderExtensionsTests()
         }
 
         [Fact]
-        public void BuilderExtensionAddsSingleSetOfServicesWhenCalledTwice()
+        public void BuilderExtensionAddsSingleSetOfServicesWhenCalledTwiceOrig()
         {
             var serviceCollection = new ServiceCollection();
-            serviceCollection.AddLogging(builder => builder.AddAzureWebAppDiagnostics(_appContext));
+            serviceCollection.AddLogging(builder => builder.AddAzureWebAppDiagnostics(_appContext, null));
             var count = serviceCollection.Count;
 
             Assert.NotEqual(0, count);
 
-            serviceCollection.AddLogging(builder => builder.AddAzureWebAppDiagnostics(_appContext));
+            serviceCollection.AddLogging(builder => builder.AddAzureWebAppDiagnostics(_appContext, null));
 
             Assert.Equal(count, serviceCollection.Count);
         }
 
         [Fact]
-        public void BuilderExtensionAddsConfigurationChangeTokenSource()
+        public void BuilderExtensionAddsSingleSetOfServicesWhenCalledTwiceOverload()
+        {
+            var serviceCollection = new ServiceCollection();
+            serviceCollection.AddLogging(builder => builder.AddAzureWebAppDiagnostics(_appContext, "customPrefix"));
+            var count = serviceCollection.Count;
+
+            Assert.NotEqual(0, count);
+
+            serviceCollection.AddLogging(builder => builder.AddAzureWebAppDiagnostics(_appContext, "customPrefix"));
+
+            Assert.Equal(count, serviceCollection.Count);
+        }
+
+        [Fact]
+        public void BuilderExtensionAddsConfigurationChangeTokenSourceOrig()
```

Comment:
```suggestion
        public void BuilderExtensionAddsConfigurationChangeTokenSource()
```
No need to add Orig at the end.